### ISSUE TITLE
Store user details in Vue Instance or add Vuex

### ISSUE THREAD

**Issue by Mondanzo** [Jugendhackt/JugendhacktApp#20]
Requesting the User object for every request is inefficent and should be stored somewhere on the device to use basic information like username and/or birthday locally in views

**Comment by LarsVomMars** [comment]
I completely agree with you, but I don't have such advanced knowledge with VueJS to do this.
Normally you should be able do store all data except the password hash which won't be send to the front end

**Comment by Mondanzo** [comment]
Marvin du bist gemein :C

**Comment by LarsVomMars** [comment]
Lol
